.github/workflows/changelog.yml

```
@@ -5,10 +5,9 @@ on:
     types: [closed]
 
 env:
-  GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+  BOT_TOKEN: ${{ secrets.BOT_TOKEN }}
```

The code looks for `GITHUB_TOKEN`

```suggestion
  GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
```